kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -181,6 +183,7 @@
 public class KafkaRequestHandler extends KafkaCommandDecoder {
     public static final long DEFAULT_TIMESTAMP = 0L;
 
+    private final Authorizer authorizer;
```

Comment:
If's better to keep the same variable order with the order that variables are initialized in the constructor, i.e. put `authorizer` after `authenticator`.